modules/amrfinder_report.nf

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process AMRFINDER_REPORT {
+    publishDir "$launchDir/$params.outdir/func_annot"
+    stageOutMode = 'copy'
+
+    container 'quay.io/microbiome-informatics/virify-python3:1.2'
+
+    input:
+        path amrfinder_tsv
+	path mobilome_gff
+
+    output:
+	path("amr_location.txt")
+
+    script:
+    if(amrfinder_tsv.size() > 0)
```

Comment:
Again, I suspect `when` is better for this

Reply:
Added!